### PR TITLE
Fix/svgs as fonts

### PR DESCRIPTION
**Pull request by meganrm**
This is not a critical change, but also shouldn't effect any of the bug testing. 

[I changed the icons on the step back and step forward buttons from images to fonts](https://aicsjira.corp.alleninstitute.org/browse/AGENTVIZ-1241). This allows us to change the color of the icon for different states, in this case, the disabled state. 

Before, there was no visual indication that the button was disabled: 
Screen Shot 2020-12-14 at 11 13 08 AM

Now, the background and the image itself is changed when disabled:
Screen Shot 2020-12-14 at 11 12 49 AM

**Pull request recommendations:**

- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-##], adds tiff file format support_
- [x] Provide description and context of changes, including any helpful screenshots.
- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Review by EricJIsaac — APPROVED** [review]

File: src/style.css [hunk 1]
```diff
@@ -87,8 +87,18 @@ canvas:focus {
     content: "\e9d9";
 }
 
+.icon-step-back:before {
+    content: "\e906";
+    color: #d8d8d8;
```
Comment (schoinh) [inline src/style.css:92]: You can use `var(--dark-theme-btn-color)` for `#d8d8d8` here and in line 97

**Review by schoinh — APPROVED** [review]
You'll have to show me how to do the Icomoon font thing sometime! Or point me to the part of the docs that you used. :)

**Review by toloudis — APPROVED** [review]
My impression is that the button background looks too "bright" to be disabled, but the implementation LGTM!

**Comment by schoinh** [comment]
> My impression is that the button background looks too "bright" to be disabled, but the implementation LGTM!

I wasn't gonna say anything because I thought it was just me, but I agree. Maybe just dimming the icon color and leaving the background color as is (or making the background a tad darker) would be better? Don't feel super strong about it 🤷‍♀️

**Review by blairlyons — APPROVED** [review]
As is it's better than no indication, but if you want to tweak the colors that's fine with me

**Comment by meganrm** [comment]
yeah, I agree, this is the color change that was also being used for the Load Model button, and for it going from purple to a gray works, but for these, I think just dimming the image is probably better

**Comment by meganrm** [comment]
Screen Shot 2020-12-16 at 5 29 45 PM

**Pull request merged**